assets/js/vue/components/search/ResultSet.vue

```diff
@@ -89,31 +92,31 @@
                         queryParam: 'status'
                     },
                     fundingCycle: {
-                        label: 'Funding Cycles',
+                        label: templateSwitch.getLabel('fundingCycle'),
                         queryParam: 'fundingCycle'
                     },
                     fundingOrg: {
                         label: 'Funding Organizations',
                         queryParam: 'fundingOrg'
                     },
                     researchGroup: {
-                        label: 'Research Groups',
+                        label: templateSwitch.getLabel('researchGroup'),
                         queryParam: 'researchGroup'
+                    },
+                    projectDirector: {
+                        label: 'Project Directors',
+                        queryParam: 'projectDirector'
                     }
                 },
-                facetCheckBoxes: {
-                    status: '',
-                    fundingOrg: '',
-                    researchGroup: ''
-                },
                 currentPage: 1,
                 perPage: this.formValues.perPage,
                 perPageOptions: [
                     { value: 10, text: '10' },
                     { value: 25, text: '25' },
                     { value: 50, text: '50' },
                     { value: 100, text: '100' }
-                ]
+                    ],
+                    showFundingCycleFacet: templateSwitch.isGrpTemplate(),
```

Comment:
I'm not a big fan of this method name. It's should probably be called showFundingCycles or something. What is the next template or another template also wants to this, then the GRP part of the method name has no meaning.